.github/workflows/workflow-distributor.yml

```
@@ -0,0 +1,22 @@
+name: "Distribute Workflows"
+on:
+  workflow_dispatch:
+  push:
+    # branches:
+    # - main
```

```suggestion
     branches:
       - main
```

to be reenabled at the end